Assets/MixedRealityToolkit.Extensions/Sharing/SpatialAlignment/Common/SpatialCoordinateServiceBase.cs

```diff
@@ -65,15 +65,20 @@ protected override void OnManagedDispose()
             knownCoordinates.Clear();
         }
 
-        /// <inheritdoc />
-        public bool TryGetKnownCoordinate(string id, out ISpatialCoordinate spatialCoordinate)
+        bool ISpatialCoordinateService.TryGetKnownCoordinate(string id, out ISpatialCoordinate spatialCoordinate)
         {
             if (!TryParse(id, out TKey key))
             {
                 throw new ArgumentException($"Id {id} is not recognized by this coordinate service.");
             }
```

Comment:
Although this was written to throw before, if we are already returning a bool, we should probably just return false here.

Reply:
I think I disagree with changing this behavior. I believe passing an invalid value as an argument warrants an exception, whereas returning true/false is an indication of whether or not that particular coordinate exists.